index.html

```diff
@@ -1,12 +1,374 @@
+                  <a class="quadcopters-link" href="catalog.html"
+                    >Фитнес-браслеты</a
+                  >
+                </li>
+              </ul>
+            </div>
+          </div>
+          <nav class="nav-page">
+            <ul class="nav-page__list">
+              <li class="nav-page__item">
+                <a class="nav-page__link" href="#">доставка</a>
+              </li>
+              <li class="nav-page__item">
+                <a class="nav-page__link" href="#">гарантия</a>
+              </li>
+              <li class="nav-page__item">
+                <a class="nav-page__link" href="#">контакты</a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </header>
+      <main class="main-page">
+        <section class="slider">
+          <h2 class="visualy-hidden">Слайдер</h2>
+          <div class="wrapper-slider">
+            <input checked type="radio" name="slider" id="slide-1" />
+            <input type="radio" name="slider" id="slide-2" />
+            <input type="radio" name="slider" id="slide-3" />
+            <button class="slider-button slider-prev" type="button">
+              <span class="visually-hidden">назад</span>
+            </button>
+            <button class="slider-button slider-prev" type="button">
+              <span class="visually-hidden">вперед</span>
+            </button>
+            <ol class="slids-list">
+              <li class="slide-item">
+                <a class="slide-link" href="#">
+                  <img class="slide" src="" alt="фото-1" />
+                </a>
+              </li>
+              <li class="slide-item">
+                <a class="slide-link" href="#">
+                  <img class="slide" src="" alt="фото-2" />
+                </a>
+              </li>
+              <li class="slide-item">
+                <a class="slide-link" href="#">
+                  <img class="slide" src="" alt="фото-3" />
+                </a>
+              </li>
+            </ol>
+            <div class="slider-info">
+              <div class="slider-desk">
+                <strong class="slider-title"
+                  >Порхает как бабочка, жалит как пчела!</strong
+                >
+                <p class="slider-text">
+                  Этот обычный, на первый взгляд, квадрокоптер оснащен мощным
+                  лазером, замаскированным под стандартную камеру.
+                </p>
+              </div>
+              <a class="btn-link" href="#">подробнее</a>
+              <div class="controls">
+                <label for="slide1"></label>
+                <label for="slide2"></label>
+                <label for="slide3"></label>
+              </div>
+              <div class="characteristic">
+                <ul class="characteristic-range-item">
+                  <li class="characteristic-range-list">
+                    <span>800 м</span>
+                    <p>Дальность полета</p>
```

Comment:
```suggestion
                    <span>Дальность полета</span>
```
короткие тексты лучше через span